python/Ganga/GPIDev/Lib/Registry/RegistrySlice.py

```diff
@@ -236,13 +236,18 @@ def select_by_range(this_id):
                                 'undefined select attribute: %s' % str(a))
                     else:
 
+                        logger.debug("Testing: %s MOO!" % a)
```

Comment:
Yes, I was trying to determine what code paths where actually being called, this can at least be trimmed down